src/App.scss

```diff
@@ -1,5 +1,354 @@
+// Fonts
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+// Colors
+$lightblue: #0290ff;
+$coolblue: #4386bf;
+$white: #fff;
+$gray: #e8e8e8;
+$black: #121212;
+
+// General styling
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
 }
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  background-color: #fafafa;
+}
+
+// Navbar styles
+.navbar {
+  display: flex;
```

Comment:
- [ ] great job . lets perfect it. please align the user place holder and the book heading like in the original desing. Great styling.
yours
![Screenshot (938)_LI](https://user-images.githubusercontent.com/65298048/229934493-91ca800c-cccd-4db5-8348-bcf897dee0fe.jpg)
original
![Screenshot (937)](https://user-images.githubusercontent.com/65298048/229934606-adac4946-a847-4747-b9af-ed344cf984a2.png)